test/support/utils.rb

```diff
@@ -28,23 +28,27 @@ def assert_fail_with_log msg, lines
                      lines.map{|l| "> #{l}"}.join
     end
 
-    def debug_code(program, **options, &block)
+    def debug_code(program, boot_options: "-r debug/run", remote_debug: true, &block)
```

Comment:
perhaps we can introduce another method like `remote_debug_code` and keep this one simple? we can also name things better in that method without causing any confusion.

Reply:
Could you tell me more about this? I don't get it.